src/style.css

```diff
@@ -0,0 +1,121 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  color: #000;
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.card {
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  width: 200px;
+  padding: 14px;
+  border-radius: 5px;
+}
+
+.card__picture {
+  margin: 0 5px;
+  margin-top: 16px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin-top: 42px;
+  font-size: 12px;
+  line-height: 15px;
+  font-weight: 500;
+}
+
+.card__code {
+  margin-top: 5px;
+}
+
+.double {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.card__feedblock {
+  display: flex;
+}
+
+.card__rating {
+  display: flex;
+  margin-top: 15px;
+}
+
+.star {
+  width: 15px;
+  height: 15px;
+  background-image: url("./images/star.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--1 .star:nth-child(-n + 1),
+.stars--2 .star:nth-child(-n + 2),
+.stars--3 .star:nth-child(-n + 3),
+.stars--4 .star:nth-child(-n + 4),
+.stars--5 .star:nth-child(-n + 5) {
+  background-image: url("./images/star-active.svg");
+}
```

Comment:
stars should be the same as in Stars task

Reply:
fixed